src/buffer/out/textBuffer.hpp

```diff
@@ -373,12 +372,15 @@ class TextBuffer final
 
     TextAttribute _currentAttributes;
     til::CoordType _firstRow = 0; // indexes top row (not necessarily 0)
+    uint64_t _mutationCount = 0;
 
     Cursor _cursor;
-
-    bool _isActiveBuffer = false;
-
     std::vector<ScrollMark> _marks;
+    // While safe RAII wrappers for URegularExpression* would improve maintainability, it would make

```

Comment:
I feel like you can use `wistd::unique_ptr` or `wil::unique_any` and only forward-declare like... two things. Is that not better than manual management?